### DISCUSSION TITLE
For Airflow 2.6.0,  I cannot get log for finished tasks. I can only get a host name in the webUI.

### DISCUSSION THREAD

**Discussion by 2470370075** [apache/airflow#31976] [Q&A]
![image](https://github.com/apache/airflow/assets/39545205/29ddf6da-5d74-4f30-ad2d-9e414430a220)

For runing tasks, i can get logs  normally.I used CeleryExecutor and multiple nodes.

I found this in file_task_handler.py. Why get log from server only when the task is in runing or deferred state?
![image](https://github.com/apache/airflow/assets/39545205/bebf215a-1e19-4d4f-b87a-48ba70024935)

**Comment by potiuk** [comment]
You need to look at your logging configuration -> what kind of logging you have etc, also look at the logs of your webserver (and potentially workers) and see if there are any errors. If you see some and your attempts to diagnose it fail - open an issue but you need to  provide all the relevant details of the configuration and the log files from webserver and potentially workers from around the time you are looking at logs.

Also I'd strongly suggest to use 2.6.1 or wait couple of hours (we are in a process of release) and use 2.6.2. Chance is that the problem has been fixed in a bugfix release. Generally it's a bad idea to try .0 release where bugfix releases are already released - you are not applying the dozens of bug fixes that have been released since .0 version.
